### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server Files_Screen Reader:Narrator is reading the content of background page after Cancel button

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#113]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0)Chrome(68.0.3440.106)
Screen Reader
MAS ID Violated: MAS 1.3.2 

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Switch on Narrator
4.Tab till Microsoft IIS  on the top navigation bar and press enter
5.Tab till Web server and press enter
6.Tab till Files and press enter
7.Tab till more command button(...) and press enter
8.Tab till Edit button and press enter
9.Tab till cancel button and check again pressing tab it is reading the background page content or not

Actual Result:
Narrator is reading the content of background page after Cancel button

Expected Result:
Narrator should not read the content of background page

User Impact:
Screen Reader users will get confused if Narrator will read the content of background page

Mas Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

Reference video:

[Bug_113_Narrator is reading content of background page.pptx](https://github.com/Microsoft/IIS.WebManager/files/2326710/Bug_113_Narrator.is.reading.content.of.background.page.pptx)

**Comment by jhkimnew** [comment]
This issue will be fixed when we use WAC SDK controls.

**Comment by jhkimnew** [comment]
This will be fixed when we use WAC SDK controls.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.2; #C+E-H; #.NETWebsites; #Chrome; #NVDA; #Narrator; #Edge

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
